packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -982,6 +982,35 @@ IWorker engine = WorkerFactory.CreateWorker(BackendType.GPUCompute, model);
 `,
 ];
 
+export const sana = (model: ModelData): string[] =>	[
```

Comment:
would it be possible to reduce the size of the snippet if possible? the idea behind snippets is that it should be the minimum amount of code required to get an inference.

perhaps we can remove the `generator` bits and assume that the model is loaded to `cuda` by default.

Reply:
Of course. Let me fix it.

Reply:
Done